.github/workflows/build-cli.yml

```
@@ -95,7 +96,7 @@ jobs:
             { build_prefix: "oss", artifact_prefix: "-oss", readable: "open source license" },
             { build_prefix: "bit", artifact_prefix: "", readable: "commercial license" }
           ]
-    runs-on: ${{ matrix.os.distro }}
+    runs-on: ${{ matrix.os.runner || matrix.os.distro }}
```

Related to the question above:

```suggestion
    runs-on: ${{ matrix.os.distro }}
```

---

thanks for the suggestion! this is much cleaner
committed and validated the builds were successful